cpu/stm32/cpu_init.c

```diff
@@ -396,13 +398,15 @@ void backup_ram_init(void)
 #define BACKUP_RAM_MAGIC    {'R', 'I', 'O', 'T'}
 #endif
 
+#if IS_ACTIVE(CPU_HAS_BACKUP_RAM)
```

Comment:
```suggestion
__attribute__((unused))
```

is IMO much better than using the preprocessor, as we get better LSP suggestions, better static analysis results and wide compile test coverage when not hiding functions from the compiler.

Reply:
Hehe, you did it yourself in https://github.com/RIOT-OS/RIOT/commit/f4c5cf173d5effc3eeb94bb2d602f22cfdc66f66